routers/web/repo/issue_content_history.go

```diff
@@ -130,6 +130,11 @@ func GetContentHistoryDetail(ctx *context.Context) {
 		return
 	}
 
+	if history.IssueID != issue.ID {
+		ctx.NotFound("GetContentHistoryDetail", nil)
+		return
+	}
```

Comment:
I do not think it is good enough.

Checking the `id` again and again, it would be still forgotten next time.

I believe the low-level functions should respect the "related ID", for example: `GetIssueContentHistoryAndPrev(ctx, historyID)` -> `GetIssueContentHistoryAndPrev(ctx, issue.ID, historyID)`, because the "issue" has been check above by `GetActionIssue`. Then there won't be low-level mistakes.